client/src/annoMatrix/loader.ts

```diff
@@ -125,26 +128,31 @@ export default class AnnoMatrixLoader extends AnnoMatrix {
     colSchema.writable = true;
     const colName = colSchema.name;
     if (
-      _getColumnSchema(this.schema, "obs", colName) ||
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-      (this as any)._cache.obs.hasCol(colName)
+      _getColumnSchema(this.schema, Field.obs, colName) ||
+      this._cache.obs.hasCol(colName)
     ) {
       throw new Error("column already exists");
     }
 
     const newAnnoMatrix = this._clone();
     let data;
     if (isArrayOrTypedArray(value)) {
+      // @ts-expect-error ts-migrate --- TODO revisit:
```

Comment:
the issue is that `isArrayOrTypedArray()` lacks a type predicate in its return type declaration.

Reply:
Fixed in #2376.